sdk/core/azure-core/azure/core/polling/base_polling.py

```diff
@@ -183,13 +187,20 @@ class OperationResourcePolling(LongRunningOperation):
      https://aka.ms/azsdk/autorest/openapi/lro-options
     """
 
-    def __init__(self, operation_location_header="operation-location", *, lro_options=None):
-        self._operation_location_header = operation_location_header
+    _async_url: str
```

Comment:
Is it optional?

I don't see we give it a default value?

Reply:
No it's not optional, if it's set it has a value